### ISSUE TITLE
NullPointerException in DockerClientFactory.dockerHostIpAddress with localstack container on 1.12.4

### ISSUE THREAD

**Issue by lanwen** [testcontainers/testcontainers-java#2120]
We use static init and catching NPE on 1.12.4 release

```
Caused by: java.lang.NullPointerException
	at org.testcontainers.DockerClientFactory.dockerHostIpAddress(DockerClientFactory.java:240)
	at org.testcontainers.containers.ContainerState.getContainerIpAddress(ContainerState.java:29)
	at org.testcontainers.containers.localstack.LocalStackContainer.configure(LocalStackContainer.java:64)
	at org.testcontainers.containers.GenericContainer.doStart(GenericContainer.java:307)
```

**Comment by bsideup** [comment]
Thanks! I submitted a PR to fix it:
https://github.com/testcontainers/testcontainers-java/pull/2119

**Comment by bsideup** [comment]
As a workaround, you can call `DockerClientFactory.instance().client()` before starting the container.

We will fix the bug ASAP and cut another release, sorry for the inconvenience!

**Comment by dparrella** [comment]
Looks like the PR with the fix was merged. Can this be closed yet? Any ETA on when the next release will be cut?

**Comment by bsideup** [comment]
@dparrella fixed in 1.12.5
